sormas-backend/src/main/java/de/symeda/sormas/backend/report/AggregateReportFacadeEjb.java

```diff
@@ -307,6 +310,85 @@ public List<AggregatedCaseCountDto> getIndexList(AggregateReportCriteria criteri
 		return resultList;
 	}
 
+	@Override
+	public List<AggregateReportDto> getDuplicateAggregateReports(AggregateReportCriteria criteria) {
```

Comment:
Since this method is used to retrieve all reports, not only duplicates, I'd rather call it `getAggregateReports`

Reply:
changed